### ISSUE TITLE
Documentation : Docker / localhost - Remote unavailable when trying to proxy localhost

### ISSUE THREAD

**Issue by ArthurD1** [msqd/harp#593]
When installing harp with docker, if proxying to a local service one has to specify `--network="host"` for the docker container to point to your docker host.

```
docker run --network="host" -it --rm -p 4000-4100:4000-4100 makersquad/harp-proxy
```

**Comment by hartym** [comment]
This is true for linux hosts, but for docker for desktop (windows, macos) machines where the underlying linux kernel is a separated virtual host, it probably does not work and host.docker.interal should probably be used instead.
